### PR TITLE
[Console] Add docs for ConsoleSignalEvent

### PR DESCRIPTION
**Pull request by sfmok**
Define the new console event `ConsoleSignalEvent` in console component docs

### CODE REVIEW

File: components/console/events.rst [hunk 1]
```diff
@@ -153,5 +153,37 @@ Listeners receive a
...
+        $signal = $event->getHandlingSignal();
+        
+        if (\SIGINT === $signal) {
+            echo "bye bye!";
+        }
+    });
+
+.. versionadded:: 5.2
+
+    The ``ConsoleEvents::SIGNAL`` event was introduced in Symfony 5.2.
```
Comment (OskarStark) [inline components/console/events.rst]:
```suggestion
    The ``ConsoleSignalEvent`` class was introduced in Symfony 5.2.
```

File: components/console/events.rst [hunk 2]
```diff
@@ -153,5 +153,37 @@ Listeners receive a
     This event is also dispatched when an exception is thrown by the command.
     It is then dispatched just after the ``ConsoleEvents::ERROR`` event.
     The exit code received in this case is the exception code.
+    
+.. _console-events-signal:
+
+The ``ConsoleEvents::SIGNAL`` Event
+--------------------------------------
```
Comment (OskarStark) [inline components/console/events.rst]: The underline should match the length of the headline

File: components/console/events.rst [hunk 3]
```diff
@@ -153,5 +153,37 @@ Listeners receive a
     This event is also dispatched when an exception is thrown by the command.
     It is then dispatched just after the ``ConsoleEvents::ERROR`` event.
     The exit code received in this case is the exception code.
+    
+.. _console-events-signal:
```
Comment (OskarStark) [inline components/console/events.rst]: This line can be removed, there is no deeplink to this section right now

**Comment by javiereguiluz** [comment]
Mokhtar, thanks a lot for contributing this feature and I'm sorry it took us so long to merge it.

**Pull request merged**